diff --git a/test/rubygems/test_bundled_ca.rb b/test/rubygems/test_bundled_ca.rb
index 6973758c4c..71f61ca8f7 100644
--- a/test/rubygems/test_bundled_ca.rb
+++ b/test/rubygems/test_bundled_ca.rb
@@ -32,7 +32,7 @@ def assert_https(host)
     http.verify_mode = OpenSSL::SSL::VERIFY_PEER
     http.cert_store = bundled_certificate_store
     http.get('/')
-  rescue Errno::ENOENT, Errno::ETIMEDOUT, SocketError
+  rescue Errno::ECONNREFUSED, Errno::ENETUNREACH, Errno::ENOENT, Errno::ETIMEDOUT, SocketError
     skip "#{host} seems offline, I can't tell whether ssl would work."
   rescue OpenSSL::SSL::SSLError => e
     # Only fail for certificate verification errors
-- 
2.29.2

